### DISCUSSION TITLE
LO-TO

### DISCUSSION THREAD

**Discussion by huan58** [nguyen-group/QERaman#19] [General]
I am wondering if the LO-TO splitting is considered in this code.

**Comment by nguyen-group** [comment]
Hi,

It might not be since LO-TO splitting is not supported in ph.x (with electron-phonon (el-ph) calculation).
If you want to go with LO-TO, you can use EPW package in QE to obtain the el-ph matrix elements with the phonon correction. Modified EPW might be required to output the full el-ph matrix elements. Then, using this matrix element for raman.x

The example can be found in ref: https://pubs.acs.org/doi/10.1021/acs.jpclett.1c03826
(It is an in-house code that we have not opened source yet.)

Best regards,
Nguyen
